Triangulation_2/include/CGAL/mark_domain_in_triangulation.h

```diff
@@ -0,0 +1,112 @@
+// Copyright(c) 2022  GeometryFactory (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Andreas Fabri
+
+#ifndef CGAL_MARK_DOMAIN_IN_TRIANGULATION_H
+#define CGAL_MARK_DOMAIN_IN_TRIANGULATION_H
+
+#include <CGAL/license/Triangulation_2.h>
+
+#include <list>
+#include <deque>
+
+#include <boost/property_map/property_map.hpp>
+
+#include <CGAL/Triangulation_2/internal/In_domain.h>
+#include <CGAL/Unique_hash_map.h>
+
+namespace CGAL {
+
+namespace internal {
+
+template <typename CDT, typename InDomainPmap>
```

Comment:
I think the algorithm applies to any constrained triangulation, not a Delunlay one so should this be CT, instead of CDT?